venv/Lib/site-packages/PIL/JpegImagePlugin.py

```
@@ -0,0 +1,902 @@
+        im, fp, [ImageFile._Tile("jpeg", (0, 0) + im.size, 0, rawmode)], bufsize
+    )
+
+
+def _save_cjpeg(im: Image.Image, fp: IO[bytes], filename: str | bytes) -> None:
+    # ALTERNATIVE: handle JPEGs via the IJG command line utilities.
+    tempfile = im._dump()
+    subprocess.check_call(["cjpeg", "-outfile", filename, tempfile])
+    try:
+        os.unlink(tempfile)
+    except OSError:
+        pass
```

Ruff (0.12.2)

Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(OSError)`

(SIM105)